docs/datactl.md

```diff
@@ -32,7 +32,7 @@ datactl [flags]
 ```
       --as string                      Username to impersonate for the operation. User could be a regular user or a service account in a namespace.
       --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
-      --cache-dir string               Default cache directory (default "/Users/ztaylor/.kube/cache")
+      --cache-dir string               Default cache directory (default "/Users/akhileshsrivastava/.kube/cache")
```

Comment:
Update this auto-generated info to not have any specific user names? - Same in other .md files

Reply:
We will handle this as part of our enhancements tracked in [ticket 25448](https://zenhub.ibm.com/workspaces/symposium-workflow-5c49e21b79c5ae72931da0d6/issues/symposium/track-and-plan/25448)